### ISSUE TITLE
Single instance prediction

### ISSUE THREAD

**Issue by rvandernoort** [ML4SE2022/group6#4]
Dear group 6, 

We were evaluating your repository for the artifact evaluation and one of the requirements was to have a single instance prediction mechanism included. Currently your README does not inform me of any way to do this, are you still working on this or do you have a way to do this already but not documented? Otherwise you might lose some points there.

Cheers,
Group 5

**Comment by RubenNair** [comment]
Hi Group 6,

Sorry for the late response! Single instance prediction can be done by changing the value of `early_eval_stop` on [line 28 in `eval-all.sh`](https://github.com/ML4SE2022/group6/blob/97fb3034a7408fe226e84549a43cdb08c572b380/eval-all.sh#L28) to `1`, and then running this command that is shown in the README (but modifying the end to only run `eval-all.sh`): 
> `docker run --gpus all --mount type=bind,source=$(pwd)/dataset--mount type=bind,source=$(pwd)/save,target=/save,target=/save --mount type=bind,source=$(pwd)/logs,target=/logs --entrypoint bash token_completion [eval-all.sh]`

Currently, the `eval-all.sh` only has the evaluation for the pretrained Python, fine-tuned JavaScript model evaluated on Python uncommented, so make sure that you have [this model](https://drive.google.com/file/d/1NpP4wdGxwnK6PbFZ90UePFD3g53c1zU_/view?usp=sharing) downloaded and put in the right location (see README), and [follow these steps](https://github.com/microsoft/CodeXGLUE/tree/main/Code-Code/CodeCompletion-token#dataset) to download and preprocess the Python dataset.

Also a small note, the predictions it makes are stored in a predictions.txt file, but we did not generate / find an answers.txt file for all datasets. Instead, we log the results (accuracy) during the evaluation, and thus this value (between 0 and 1) can be found at the end of the generated log file.

Hope this helps,
Group 6
